src/fluree/db/query/history.cljc

```diff
@@ -298,12 +307,15 @@
            assert-key          (json-ld/compact const/iri-assert compact)
            retract-key         (json-ld/compact const/iri-retract compact)
            data-key            (json-ld/compact const/iri-data compact)
-           commit-key          (json-ld/compact const/iri-commit compact)]
-
-       (-> {commit-key commit-wrapper}
-           (assoc-in [commit-key data-key] commit-meta)
-           (assoc-in [commit-key data-key assert-key] asserts)
-           (assoc-in [commit-key data-key retract-key] retracts)))
+           default-ctx-key     (json-ld/compact const/iri-default-context compact)
+           commit-key          (json-ld/compact const/iri-commit compact)
+           jld                 (-> {commit-key commit-wrapper}
+                                   (assoc-in [commit-key data-key] commit-meta)
+                                   (assoc-in [commit-key data-key assert-key] asserts)
+                                   (assoc-in [commit-key data-key retract-key] retracts))]
+       (if default-ctx
```

Comment:
Do we still need all the `default-ctx` stuff in here now that we're not calling it from the `default-context` method? The contract of the `commit-details` functionality was to return the commit, and the commit doesn't have the context inline.

Reply:
AFAIUI the commit not being inline is purely a storage optimization. So the in-memory commit data structure very well could and perhaps should have it inline. But you're likely correct that this change isn't needed any longer to enable this feature. If so I'll revert it.

Reply:
It was no longer needed and is now reverted. Good catch!

Reply:
Not only a storage optimization (e.g. the schema.org context might be larger than some databases!), but using content-addressing, if thousands of ledgers used schema.org, that is is just one file shared amongst all of them. So data reusability is another benefit.